client/src/FileUtils.lua

```diff
@@ -333,16 +334,105 @@ function fileUtils.write(path, filename, data)
   end
 end
 
+local function prettifyJson(jsonString)
```

Comment:
What does this function offer that setting the indent flag of our json library doesn't?